include/acornfs.h

```diff
@@ -45,7 +45,7 @@ typedef struct _tag_ACORN_DIRECTORY {
   char * name;
   uint8_t options;
   int num_of_files;
-  ACORN_FILE files[];
+  ACORN_FILE files[1];
```

Comment:
Doesn't this change the size of ACORN_DIRECTORY? I.e. the structure now includes 1 element of ACORN_FILE, where as before it was only the size of the header? This may not be a problem I'm just trying to understand why the change.

Reply:
Oh, yes, it does. And I never corrected for that. I guess that means we return a buffer a little too big.
The reason was that the C89 compiler I'm using doesn't support the unspecified array size within a struct - it gives an error. So I just made it use 1 and intended to reduce the size of the allocated space, but then forgot. It's not a huge problem that the buffer's a little larger, but if you want I can fix that up in the rest of the code?